src/pages/organize/[orgId]/people/lists/[viewId]/index.tsx

```diff
@@ -37,7 +38,13 @@ export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
     // Check if user is an official
     // TODO: Consider moving this to some more general-purpose utility
     const officialMemberships = await getUserMemberships(ctx, false);
-    if (!officialMemberships.includes(parseInt(orgId as string))) {
+
+    const isSuperUser = await getSuperUser(ctx, false);
+
+    if (
+      !officialMemberships.includes(parseInt(orgId as string)) &&
+      !isSuperUser
+    ) {
```

Comment:
I really like when conditions are named, it makes it so much easier to read the code. I suggest to both save `!officialMem....` in a named constant, and then save the "sum" of that constant and `isSuperUser` in a constant. That means something like: 
```
const isOfficialMember = officialMemeber....;
const isSuperUser = ctx....;
const notAllowedAccess = !isOfficialMember && !isSuperUser; 

if(notAllowedAccess) {....
```